### ISSUE TITLE
Canvas is not a constructor

### ISSUE THREAD

**Issue by sgraham785** [lokesh/color-thief#152]
In node v11.6.0 on macOS Mojave, I get the error 

```
Canvas is not a constructor at new CanvasImage (/Users/sgraham/Development/golang/src/github.com/permanencelabs/scrape/node_modules/color-thief/js/color-thief.js:42:19)
```

after some investigation, it appears that Canvas has changed it api which no longer uses the `new Canvas()` object and now uses `Canvas.createCanvas()`

so line 42 of `color-thief.js` should look like...

```
this.canvas = Canvas.createCanvas()
```
this fix allow for me to run Color thief without issue. I can submit a PR

<img width="229" alt="Screenshot 2019-06-01 14 06 53" src="https://user-images.githubusercontent.com/3058459/58753683-197bda80-8478-11e9-8113-93859f107307.png">

**Comment by sgraham785** [comment]
@lokesh sorry i just realized that I had installed the `null2` version which has this issue... closing

**Comment by lokesh** [comment]
Just in case someone comes across this, I've updated the README.md to note that the name of the package is `colorthief`, not `color-thief`. I've contacted the owners of the `color-thief` npm package and am hoping for response so we can come up with a plan to reintegrate to prevent confusion.

Last thing, I've added Node support to Color Thief. You don't need to use the null2 fork which requires some heavy, non-JS deps. Node support was added in v2.3.0:
https://github.com/lokesh/color-thief/releases

```
const { resolve } = require('path');
const ColorThief = require('colorthief');
const img = resolve(process.cwd(), 'night.jpg');

ColorThief.getColor(img)
	.then((color) => {
		console.log(color);
	})
```
